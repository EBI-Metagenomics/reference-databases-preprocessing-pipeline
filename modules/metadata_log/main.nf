
import groovy.json.JsonOutput

process METADATA_LOG {

    label 'light'

    output:
    path("log.txt"), emit: log

    """
    dt=\$(date)
    echo 'params:\t${JsonOutput.toJson(params)}' > log.txt
    echo "date:\t\$dt" >> log.txt
    """

}