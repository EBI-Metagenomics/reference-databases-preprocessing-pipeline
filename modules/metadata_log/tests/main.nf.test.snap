{
    "test_param": {
        "content": [
            true
        ],
        "timestamp": "2024-01-04T12:30:10.749712914"
    }
}